### ISSUE TITLE
Question or new feature request.

### ISSUE THREAD

**Issue by Kerberos007** [pmorissette/ffn#133]
ffn.core.calc_erc_weights(returns, initial_weights=None, risk_weights=None, covar_method='ledoit-wolf', risk_parity_method='ccd', maximum_iterations=100, tolerance=1e-08)

great financial features.  Kudos to the great work.

one question:  is there an API similar to the above ere_weights that can do 
1 obtain the optimum weights of multiple assets
2 goal = to minimize the portfolio variance (not the same as mean-variance optimization)
3 constraint = each asset contributes to the same marginal risk = risk parity

in other words: minimize portfolio Variance with Risk Parity constraint.
this would be a great addition to your already excellent package.
Thanks in advance.  keep up the good work
